apps/governance/src/routes/proposals/components/proposal-volume-discount-program-details/proposal-volume-discount-program-details.tsx

```diff
@@ -87,7 +87,7 @@ export const ProposalVolumeDiscountProgramDetails = ({
                 .map((benefitTier, index) => (
                   <div className="mb-4" key={index}>
                     <h4 className="font-semibold uppercase">
-                      Tier {index + 1}
+                      Tier {benefitTiers.length - index}
```

Comment:
The tiers here are sorted in ascending order, so the initial way of numbering was correct.
```suggestion
                      Tier {index + 1}
```